topo_processor/cog/command.py

```diff
@@ -23,11 +23,11 @@ def __init__(self, command: str, docker_ref: CommandDocker = None):
             self.container = docker_ref.get("container", None)
             self.container_tag = docker_ref.get("tag", None)
 
-    def append_arg(self, *args: str) -> "Command":
+    def append_arg(self, *args: str):
```

Comment:
these should `return self` and have the types left on.

Reply:
why does it need to return self?

Reply:
It's so you can do something like


```
Command("foo").arg(bar).arg(baz).run()
```

Reply:
i see